clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2846,10 +2848,13 @@ The ``[[intel::ii]]`` attribute spelling is a deprecated synonym for
     [[intel::initiation_interval(4)]] for (int i = 0; i < 10; ++i) var++;
   }
 
+  [[intel::initiation_interval(4)]] void foo1 { }
+
   template<int N>
   void bar() {
     [[intel::initiation_interval(N)]] for(;;) { }
   }
+  [[intel::initiation_interval(N)]] void bar1 { }
```

Comment:
Minor nit:  Please introduce the template argument N.

Reply:
Done. Thank you.